ebcl/tools/initrd/initrd.py

```diff
@@ -323,6 +323,9 @@ def create_initrd(self) -> Optional[str]:
 
         # Create lib and bin folder symlinks
         self.config.fake.run_sudo(f'ln -sf usr/lib {self.target_dir}/lib')
+        self.config.fake.run_sudo(f'ln -sf usr/lib32 {self.target_dir}/lib32')
+        self.config.fake.run_sudo(f'ln -sf usr/lib64 {self.target_dir}/lib64')
+        self.config.fake.run_sudo(f'ln -sf usr/libx32 {self.target_dir}/libx32')
```

Comment:
/lib64 should be enough. The other two are for 32 bit x86_64 binaries and 32 bit i386 binaries.
I don't think anyone will ever use such a binary in the initrd or even on the rootfs.

Apart from that I wonder if we do want to create the symlink as well for arm64, because usually there is no /lib64 (e.g. libc just uses `/usr/lib/ld-linux-aarch64.so.1` (https://packages.ubuntu.com/noble/arm64/libc6/filelist).
Other architectures like risc v will also probably not use the other directories, because it is a backward dependency layer and that backward dependency does not exist for arm (although technically there is armhf, but there is no backward compatibility layer in linux distros for some reason) or rsic v